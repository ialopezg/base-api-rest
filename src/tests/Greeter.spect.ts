import { Greeter } from '../index';

it('My Greeter', () => {
  const tree = Greeter('John Doe').toString();

  expect(tree).toMatchSnapshot();
});
