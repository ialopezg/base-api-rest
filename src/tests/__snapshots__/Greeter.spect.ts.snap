// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Greeter 1`] = `"Hello John Doe"`;
